webapp/src/components/modals/create_issue/create_issue.test.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import CreateIssueModal from './create_issue';
+
+jest.mock('utils/user_utils', () => ({
+    getErrorMessage: jest.fn(() => 'Error occurred'),
+}));
+
+describe('CreateIssueModal', () => {
+    const defaultProps = {
+        close: jest.fn(),
+        create: jest.fn(() => Promise.resolve({})),
+        post: null,
+        title: '',
+        channelId: '',
+        theme: {
```

Comment:
I think we should also test the theme changes if possible

Reply:
I tried adding them but was unable to do so